autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -295,7 +295,8 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
                'XT': {'n_estimators': 300},
                'KNN': {},
                'custom': ['GBM'],
-             }
+                # 'LR': {'C': 1}, # Enable linear models
```

Comment:
can you update fit() documentation with 'LR' option, similar to the other sklearn models.

Reply:
I removed the line for now.